packages/frontend/app/components/school-competencies-expanded.js

```diff
@@ -1,27 +1,27 @@
 import Component from '@glimmer/component';
 import { service } from '@ember/service';
 import { all } from 'rsvp';
-import { tracked } from '@glimmer/tracking';
+import { cached, tracked } from '@glimmer/tracking';
 import { action } from '@ember/object';
-import { dropTask, restartableTask } from 'ember-concurrency';
+import { dropTask } from 'ember-concurrency';
 import { filterBy, uniqueValues } from 'ilios-common/utils/array-helpers';
+import { TrackedAsyncData } from 'ember-async-data';
 
 export default class SchoolCompetenciesExpandedComponent extends Component {
   @service store;
   @tracked competenciesToAdd = [];
   @tracked competenciesToRemove = [];
-  @tracked schoolCompetencies;
```

Comment:
needs to be replaced with a getter that wraps around `get schoolCompetenciesData()`.